### PR TITLE
Bttribev2 10320 progress tiny and result indictor

### PR DESCRIPTION
**Pull request by AdimAnyvision**
## What I did
Added progress text + tiny progress
Added ResultIndicator for user feedback icons for success, error, in queue

## How to test
Run storybook and see Progress examples and ResultIndicator

### Checklist
- [ ] Does this need a new/update storybook example?
- [ ] Does this need new/update tests?

- [ ] run eslint
- [ ] run test
- [ ] run build

If your answer is yes to any of these, please make sure to include it in your PR.

> NOTE: Please submit all PRs to the `development` branch.

### CODE REVIEW

File: src/components/ProgressWithIndicator/ProgressWithIndicator.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+ProgressWithIndicator.propTypes = {
+  /** For css customization in progress. */
+  className: propTypes.string,
+  /** The progress variant. */
+  variant: propTypes.oneOf(['linear', 'circle']),
+  /** Progress value between - <code>0-100</code>. */
+  value: propTypes.number,
+  /** Determines if text with `processing` or not. */
+  tiny: propTypes.bool,
```
Comment (ozAnyvision) [inline src/components/ProgressWithIndicator/ProgressWithIndicator.js:61]:
```suggestion
  isTiny: propTypes.bool,
```

File: src/components/ProgressWithIndicator/ProgressWithIndicator.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  /** Determines if the action was done successfully. */
+  success: propTypes.bool,
+  /** Determines if the action fails. */
+  error: propTypes.bool,
+  /** Message to show when action fails.*/
+  errorMessage: propTypes.string,
+  /** Message to show when action succeeds.*/
+  successMessage: propTypes.string,
+  /** Action waits to be done.*/
+  inQueue: propTypes.bool,
```
Comment (ozAnyvision) [inline src/components/ProgressWithIndicator/ProgressWithIndicator.js:71]: rename?

File: src/index.js [hunk 3]
```diff
@@ -26,5 +26,7 @@ export { Banner } from './components/Banner'
 export { Progress } from './components/Progress'
 export { Tabs } from './components/Tabs'
 export { RangeSlider } from './components/RangeSlider'
+export { ResultIndicator } from './components/ResultIndicator'
```
Comment (ozAnyvision) [inline src/index.js:29]: do we want to export that?

**Review by ozAnyvision — CHANGES_REQUESTED** [review]

File: src/components/ResultIndicator/ResultIndicator.module.scss [hunk 4]
```diff
@@ -0,0 +1,71 @@
+@import '~@anyvision/style-guide';
+
+@keyframes expandIn {
+  0% {
+    opacity: 0;
+    transform: scale(1.2);
+  }
+  90%{
```
Comment (IrenaSAnyVision) [inline src/components/ResultIndicator/ResultIndicator.module.scss:8]:
```suggestion
  90% {
```

**Review by ozAnyvision — APPROVED** [review]

**Pull request merged**